internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -44,10 +43,18 @@ import (
 
 //go:generate mockgen -destination=../../../mocks/mock_quick_start.go -package=mocks github.com/mongodb/mongodb-atlas-cli/internal/cli/atlas/quickstart Flow
 
-const quickstartTemplate = `
-Now you can connect to your Atlas cluster with: mongosh -u %s -p %s %s
+const quickstartTemplateMongoshDetected = `
+MongoDB Shell detected. Connecting to your Atlas cluster:
+$ mongosh -u %s -p %s %s
+`
 
+const quickstartTemplateMongoshNotDetected = `
+MongoDB Shell not detected. To install, open www.mongodb.com/try/download/shell
```

Comment:
[q] Like how we prompt for `atlas login`, do we want to open the browser on the users behalf here?

(The more I think about it, that's not great, as we don't wait for the user to download and try again.)

Reply:
I think installing shell is more of an optional choice for users, not a requirement. This is is different from login, since users might already have Compass (and Compass has MongoDB shell in it), so download shell is not a blocker for them to do anything at this stage. CC @boooczek